Packs/Jizo_NDR/Classifiers/classifier-Jizo_NDR_Incoming_Mapper.json

```diff
@@ -0,0 +1,34 @@
+{
+	"feed": false,
+	"id": "Jizo NDR - Incoming Mapper",
+	"locked": false,
+	"logicalVersion": 14,
+	"mapping": {
+		"Jizo Alert": {
+			"dontMapEventToLabels": false,
+			"internalMapping": {
+				"Jizo Alert Category": {
```

Comment:
Per my suggestions, I think the best will be mapping to our current OOTB fields instead of creating duplicated ones only for your pack :)

Reply:
done